Pearson-Advance/catalog-plugin
# feat: add original FixedCatalog.

## Description

This PR adds the original FixedCatalog model.

## Changes made

- [x] Add FixedCatalog

## How to test

- Start environment
- Install catalog-plugin
- Run migrations
- Verify all models

## Reviewers
